packages/web-components/fast-foundation/src/button/button.template.ts

```diff
@@ -45,7 +45,7 @@ export const ButtonTemplate = html<Button>`
     >
         ${startTemplate}
         <span class="content" part="content">
-            <slot></slot>
+            <slot ${ref("content")} @slotchange="${x => x.handleContentChange()}"></slot>
```

Comment:
The slotted directive may be more beneficial here for us - it will return an array of all slotted nodes. We ultimately want to apply a class in the event that the array has a single SVG in it.